src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -41,17 +107,23 @@ public void keyServerSetup(String keyServerIp, String keyServerHttpPort) {
     this.keyServerHttpPort = keyServerHttpPort;
   }
 
+  /**
+   * Setup the server with use.sh
+   */
   private void keyServerSetup() {
-    // Setup the server with use.sh
-    executeP2pScript("use.sh", keyServerIp, keyServerHttpPort);
+    keyServerSetup(keyServerIp, keyServerHttpPort);
+
+    String command = pathToResourceDocker + "/use.sh " + keyServerIp + " " + keyServerHttpPort;
```

Comment:
if you take a look at my previous implementation with the ProcessExecutor, maybe it is possible to have something similar to executeP2pScript here? so we can just pass as an argument the script to be used (in this case use.sh) and the parameters as a string? I think this would make it less error prone since if a space gets deleted or someone forgets the / it would not run. Also it makes it easier in the future to change things around I think. What do you think?